docs/baking.md

```diff
@@ -105,125 +101,141 @@ In order to import the snapshot, run the following command:
+This service will trigger the following services to start:
+* `tezos-node-<network>.service`
+* `tezos-baker-<proto>@<network>.service`
+* `tezos-endorser-<proto>@network.service`
+
+Once services have started, you can check their logs via `journalctl`:
 ```
-sudo systemctl start tezos-baker-<proto>.service
-sudo systemctl start tezos-endorser-<proto>.service
+journalctl -f _UID=$(id tezos -u)
 ```
+This command will show logs for all services that are using `tezos` user.
 
-If the node isn't bootstrapped yet, the baker and endorser daemons will wait for it to bootstrap.
+You'll see the following messages in the logs in case everything has started
+successfully:
+```
+Baker started.
 
-Note that if you're baking with the ledger key, you should have the Tezos Baking app open.
+Endorser started.
+```
 
-Once the services are started, you can check their logs via `journalctl`:
+To stop the baking instance run:
 ```
-journalctl -u tezos-baker-<proto>.service
-journalctl -u tezos-endorser-<proto>.service
+sudo systemctl stop tezos-baking-<network>.service
 ```
 
-If everything was set up correctly, you shouldn't see any errors in the logs.
+## Running baking instance on edo2net
+
+TL;DR, in order to run baking instance on edo2net you should do the following:
 
-Logs for successfully started baker service should begin with:
+1) Install `tezos-baking` package following either [Ubuntu](#ubuntu) or [RaspberryPi OS](#pios)
+instructions.
+
+2) Run following commands:
 ```
-Node is bootstrapped.
-Baker started.
+snapshot_file=/tmp/tezos-edo2net.rolling
+wget https://edo2net.xtz-shots.io/rolling -O "$snapshot_file"
+sudo -u tezos tezos-node-edo2net snapshot import "$snapshot_file"
+
+sudo systemctl start tezos-node-edo2net
+
+sudo -u tezos tezos-client bootstrapped
+sudo -u tezos tezos-client import secret key baker <secret-key>
+sudo -u tezos tezos-client register key baker as delegate
+
+sudo systemctl start tezos-baking-edo2net
+sudo systemctl enable tezos-baking-edo2net
 ```
 
-Logs for successfully started endorser service should begin with:
+## Advanced baking instance setup
+
+### Using different data directories and node RPC address
+
+In case you want to use a different `tezos-client` data directory or RPC address,
+you should update `/etc/default/tezos-baking-<network>` file, e.g.:
 ```
-Node is bootstrapped.
-Endorser started.
+sudo vim /etc/default/tezos-baking-<network>
+```
+
+In case you want to use a different `tezos-node` data directory, you
+should edit the service configuration. To edit the service configuration, run:
 ```
+sudo systemctl edit --full tezos-node-<network>.service
+```
+
+### Using different account alias for baking
+
+In case you want to use different alias for the baking account, replace `baker` with
+the desired alias in sections about [importing](#import) and [registering](#registration)
+baker key. Also, you'll need to update `BAKER_ADDRESS_ALIAS` in
+`/etc/default/tezos-baking-<network>`.
```

Comment:
The `BAKER_ADDRESS_ALIAS` seems to be left as an after-thought, I suggest:
```suggestion
In case you want to use a different alias for the baking account:
1. replace `baker` with he desired alias in the sections about [importing](#import)
    and [registering](#registration) the baker key.
2. update the `BAKER_ADDRESS_ALIAS` by editing the
    `/etc/default/tezos-baking-<network>` file.
```